calcom/cal.com
# Seats events send email to attendees and organizer

## What does this PR do?



This PR fixes the bug where on event types with seat types, when more than 1 attendee is added an email was not sent. Now any new attendees will receive a different email.

Organizer email when a new attendee is added to the booking
![image](https://user-images.githubusercontent.com/65426560/176723130-68ec0f1b-2dd9-4428-aad7-3b08cf55dce7.png)


Fixes # (issue)



**Environment**: Staging(main branch) / Production

## Type of change




## How should this be tested?



- Create an event type with seats
- Create a booking on that event type
- Book another seat on the previous booking
	-  The attendee and organizer should receive an email

## Checklist


- I haven't added tests that prove my fix is effective or that my feature works
- I haven't checked if new and existing unit tests pass locally with my changes
